assets/built-in/transformers/dockerfile/dockerimagebuildscript/templates/buildandpushimages_multiarch.bat

```diff
@@ -29,6 +29,10 @@ if not %basename% == "scripts" (
 REM go to the parent directory so that all the relative paths will be correct
 cd {{ .RelParentOfSourceDir }}
 
+SET CONTAINER_RUNTIME={{ .ContainerRuntime }}
```

Comment:
Similar to the `buildimages.bat` and `pushimages.bat` file, we should now also allow the user to choose the container runtime while executing the `buildandpushimages_multiarch.bat` file.

https://github.com/konveyor/move2kube/blob/565394bf94dcbab7ea340b7daf603f8dee326a14/assets/built-in/transformers/dockerfile/dockerimagebuildscript/templates/buildimages.bat#L18
 
https://github.com/konveyor/move2kube/blob/565394bf94dcbab7ea340b7daf603f8dee326a14/assets/built-in/transformers/dockerfile/dockerimagebuildscript/templates/buildimages.bat#L29
 
https://github.com/konveyor/move2kube/blob/565394bf94dcbab7ea340b7daf603f8dee326a14/assets/built-in/transformers/containerimagespushscript/templates/pushimages.bat#L19
 
https://github.com/konveyor/move2kube/blob/565394bf94dcbab7ea340b7daf603f8dee326a14/assets/built-in/transformers/containerimagespushscript/templates/pushimages.bat#L23

Reply:
Then we can also add an example on how to specify the container runtime (along with registry namespace/registry url/platforms) while executing the file .

https://github.com/konveyor/move2kube/blob/565394bf94dcbab7ea340b7daf603f8dee326a14/assets/built-in/transformers/dockerfile/dockerimagebuildscript/templates/buildandpushimages_multiarch.bat#L19

Reply:
Thank you, I have made the changes.